src/extension.ts

```diff
@@ -1737,6 +1738,27 @@ const debugAttachKubernetes = async (explorerNode: ClusterExplorerResourceNode)
     }
 };
 
+const selectPodKubernetes = async () => {
```

Comment:
I've been gradually changing these `const foo = async () => ...` to `async function foo()` - would suggest adopting that new style unless the old one is specifically needed.

Reply:
> 
> 
> Thanks heaps for this @loligans! Re docs, might also be good to add some docs to the debug documentation (https://github.com/Azure/vscode-kubernetes-tools/blob/master/debug-on-kubernetes.md) as that is one of the main scenarios for this 

Yes that is the main use for this command. I haven't much time lately as I have a baby that is due in the next few days. I will try to do this today or tomorrow.